enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -449,9 +451,13 @@ func NewProvider(env environment.Env, buildRoot, cacheRoot string) (*Provider, e
 		return nil, status.WrapError(err, "enable masquerading")
 	}
 
+	networkPool := networking.NewVMNetworkPool(*netPoolSize)
```

Comment:
Should we set this to nil if pool size is 0? Otherwise we'll still try to do things like deactivate the network before adding to the pool (which will then fail, because there's no capacity)

Reply:
good point, done